packages/controller-utils/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.3]
```

Comment:
It looks like there is a change missing from this changelog: https://github.com/MetaMask/core/pull/4030

It wasn't mentioned in the PR description, but that PR adds a new constant.

Reply:
Probably a good idea to mention this as a change as well: https://github.com/MetaMask/core/pull/4084

We don't typically include dev dependency updates in changelogs, but a TypeScript bump does change the output in ways that may be important for consumers (e.g. if there was a bug fix/enhancement to `tsc` that impacts the compiled JS).

Reply:
Given the addition of that constant, a minor bump seems more appropriate for this release I think